components/encryption/src/encrypted_file/mod.rs

```diff
@@ -16,7 +16,7 @@ use crate::Result;
 mod header;
 pub use header::*;
 
-pub const TMP_FILE_SUFFIX: &str = ".tmp";
+pub const TMP_FILE_SUFFIX: &str = "tmp";
```

Comment:
Can you add a test case for this change?

Reply:
This is just a small fix, `set_extension` automatically add a `.`